src/components/media-container/usePipHook.ts

```diff
@@ -94,10 +94,10 @@ export const usePipHook = ({ isPlayerReady }: UsePipHookProps): UsePipHook => {
 			return;
 		}
 		if (!isPip && !isVisibleFromScrollingTop) {
-			requestPip?.();
+			setTimeout(requestPip, 1);
```

Comment:
Pls add a comment why there is a timeout. (I always find these things suspicious when reading through code - good if it's explained)